pants

```diff
@@ -126,6 +115,12 @@ EOF
 # are installed and up to date.
 
 function determine_pants_version {
+
+  if [ -n "${PANTS_SHA:-}" ]; then
+    get_version_for_sha "$PANTS_SHA"
```

Comment:
Why retrieve the version if it won't be used anywhere?

Reply:
This function "returns" it, by virtue of echoing it. Yay bash.

Reply:
Doh, and I am the one who wrote this function in the version in the toolchain repo ...

@benjyw: Probably worth a comment since this usage confused even me the original author ...

Reply:
Done.